features/wishlist.feature

```diff
@@ -102,6 +107,13 @@ Scenario: Update a Wishlist
     And I press the "Search" button
     Then I should see Name = "before today", Customer ID = "128" in the results
 
+Scenario: List all Wishlists
```

Comment:
Please make this in a separate PR as it is linked to a different story

Reply:
(See above comment)

Reply:
There were two stories for two endpoints. Let it be. We'll delete the get index story